docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

```diff
@@ -3,19 +3,39 @@
 include::_attributes.adoc[]
 :categories: security,web
 
-The Quarkus Security framework supports multiple authentication mechanisms, which you can use to secure your applications. 
-Some of the supported authentication mechanisms are built into Quarkus and some require you to add an extension.
-You can also combine authentication mechanisms. 
+The Quarkus Security framework supports multiple authentication mechanisms, which you can use to secure your applications.
+Some supported authentication mechanisms are built into Quarkus, and some require you to add an extension.
+You can also combine authentication mechanisms.
 
 Before you choose an authentication mechanism for securing your Quarkus applications, review the information provided.
 
+.Authentication requirements and mechanisms
+[source, adoc]
+|===
+^|Authentication Requirement ^| Authentication Mechanism
+
+^|Username and password ^|xref:security-basic-authentication-concept.adoc[Basic], xref:security-authentication-mechanisms-concept.adoc#form-auth[Form]
+
+^|Bearer Access Token  ^a| xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer], xref:security-jwt.adoc[JWT], xref:security-oauth2.adoc[OAuth2]
+
+* See the <<table>> table below.
```

Comment:
In the renderend output, this is the only table entry that has also has an entry in the Notes column. When this column is blank for all the other items in the table, it looks a bit funny and also might pose an issue for Accessibility.  Would it be possible to remove the Notes column and move this one item underneath the table (perhaps as a callout)

Reply:
Hello Sheila @sheilamjones . Rendered output was just for content moving. It is very obsolete right now. You can have a new rendered view when you go: Files changed -> then in the right upper corner -> ... -> View file.
Please, always use the last one.
Adding it here:
https://github.com/quarkusio/quarkus/blob/d0f419683041593c0a792311fdf37149083833a3/docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

Reply:
@sberyozkin @sheilamjones Hello. Please, where exactly in the latest build, which is the only relative built when providing a review, do you see that extra Notes column? :(
![image](https://user-images.githubusercontent.com/48474054/215998167-09190b4b-0d80-47ea-bf62-255d3884f32c.png)